.github/ISSUE_TEMPLATE/feature--specviz2d.md

```diff
@@ -0,0 +1,29 @@
+---
+name: 'Feature request: Specviz2D'
+about: Suggest an idea to improve Specviz2D
+title: "[FEAT] Specviz2D:"
+labels: feature, specviz2d
+assignees: ''
+
+---
+
+<!-- This comments are hidden when you submit the issue,
```

Comment:
would now be a good excuse to fix this across all issue templates?
```suggestion
<!-- These comments are hidden when you submit the issue,
```

Reply:
Sure. Done. Good catch!